python/jupytergis_lab/style/base.css

```
@@ -417,6 +417,12 @@ div.jGIS-toolbar-widget > div.jp-Toolbar-item:last-child {
   height: 600px;
   width: 100%;
 }
+
+/* Support output views, created with "Create new view for cell output" context menu */
+.jp-LinkedOutputView .jupytergis-notebook-widget {
+  height: 100%;
+}
```

```suggestion
.jp-LinkedOutputView .jp-OutputArea-child:only-child {
   height: 100%;
}
```
Making this comment to prevent getting it merged while it's not ready
Suggested by @SylvainCorlay

---

This doesn't work for me, the widget is 600px in size with this CSS. I'm also concerned about whether this CSS would apply to other widgets the user might be using in a LinkedOutputView!

---

The rule above should be done in JupyterLab core.